### ISSUE TITLE
read battery level

### ISSUE THREAD

**Issue by MadMonkey87** [marcelm/radonwave#9]
Is there any chance we can read the battery level?

**Comment by marcelm** [comment]
I don’t think that information is available. I just went through all the information (the "characteristics") offered via Bluetooth again, and apart from the things that the script already gets, there are only things like model name, firmware version and also the current time, so everything that’s interesting is already retrieved.

Since the battery level isn’t available via the app (or have you found it anywhere?), it is quite possible that the device itself doesn’t track the battery level.

**Comment by MadMonkey87** [comment]
I went trough the charactristics as well and could not find anything, but it must be possible somehow as it is shown in the app!

**Comment by marcelm** [comment]
Where do you see that? I couldn’t find it in the app.

**Comment by MadMonkey87** [comment]
![Screenshot_20210125-123255_Airthings](https://user-images.githubusercontent.com/29612400/105700715-84a9d880-5f09-11eb-8625-b415c14ee616.png)

**Comment by marcelm** [comment]
Ah, you have the Wave Plus. That’s probably why you see the battery level. I only have the regular Wave, and it doesn’t show this in the app.
